lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -523,6 +524,7 @@ dictionary ListPaymentsRequest {
     u32? offset = null;
     u32? limit = null;
     ListPaymentDetails? details = null;
+    PaymentState? state = null;
```

Comment:
We have a variant of this added to `rt-sync` branch already, but can use multiple states to filter. Maybe you can cherry pick it out: https://github.com/breez/breez-sdk-liquid/blob/7c410d4056f0edbe1f736ad56e9cb879180a8948/lib/bindings/src/breez_sdk_liquid.udl#L522

Reply:
Cool, thanks. Cherry-picked here [d1046d4](https://github.com/breez/breez-sdk-liquid/pull/615/commits/d1046d4016620aaa0bf5c6d580337bb08cc5cf04)